cfme/configure/configuration/__init__.py

```diff
@@ -77,7 +77,8 @@
         ("storage_metrics_coordinator", CFMECheckbox("server_roles_storage_metrics_coordinator")),
         ("storage_inventory", CFMECheckbox("server_roles_storage_inventory")),
         ("vmdb_storage_bridge", CFMECheckbox("server_roles_vmdb_storage_bridge")),
-
+        # NEW ANSIBLE ROLE
```

Comment:
personally don't like comments like 'new' - consider 'Ansible role added in euwe'?